dc-cudami-server/dc-cudami-server-webapp/src/main/java/de/digitalcollections/cudami/server/controller/identifiable/entity/V3DigitalObjectController.java

```diff
@@ -89,7 +90,7 @@ public V3DigitalObjectController(
           int pageSize,
       @Parameter(
               name = "active",
-              description = "the set to true, only active subcollections are returned.",
+              description = "the set to true, only active collections are returned.",
```

Comment:
```suggestion
              description = "if set to true, only active collections are returned.",
```
rather "if", or what should that "the" mean there?